index.rst

```diff
@@ -111,7 +111,7 @@ Documentación del lenguaje
 
 A continuación, primero veremos un :ref:`contrato inteligente sencilo <simple-smart-contract>` escrito en Solidity, seguido de una introduccin sobre :ref:`blockchains <blockchain-basics>` y sobre la :ref:`Máquina Virtual de Ethereum <the-ethereum-virtual-machine>`.
```

Comment:
I don't know why this line is failing. it seems that there is some kind of problem with the simple-smart-contract link, but to me it seems fine.